static/data/translation-table.json

```
@@ -318,7 +319,7 @@
     },
     {
       "cn": "豇豆",
-      "en": "cowpea",
+      "en": "Chinese long string bean",
```

```suggestion
      "en": "cowpea",
```
这个不用改吧，都是 Vigna unguiculata